aocd/__init__.pyi

```diff
@@ -0,0 +1,32 @@
+import typing as t
+
+from aocd import cli, cookies, exceptions, get, models, post, runner, transforms, utils, version
+from aocd.exceptions import AocdError, PuzzleUnsolvedError
+from aocd.get import get_data
+from aocd.post import submit
+from aocd.utils import AOC_TZ
+from aocd.version import __version__
+
+__all__ = [
+    "cli",
+    "cookies",
+    "exceptions",
+    "get",
+    "models",
+    "post",
+    "runner",
+    "transforms",
+    "utils",
+    "version",
+    "AocdError",
+    "PuzzleUnsolvedError",
+    "get_data",
+    "submit",
+    "AOC_TZ",
+    "__version__",
+    "data",
+]
```

Comment:
I'm not too familiar with .pyi files, but does it mean I now need to maintain this list in 2 different places?  That's a little annoying..

Reply:
Unfortunately, yes.

Aside from types directly in `.py` files (which is unpleasant when you need Python 2 support, and (afaik) not possible when you're "hacking" `sys.modules` like `aocd/__init__.py` does, I don't know of anything better.